tests/trainer/logging_/test_eval_loop_logging.py

```diff
@@ -616,3 +617,17 @@ def validation_step(self, batch, batch_idx):
     )
 
     trainer.fit(model)
+
+
+@RunIf(min_gpus=1)
+def test_move_metrics_to_cpu(tmpdir):
```

Comment:
Why did you put this in `eval_loop_logging` if it's training related and not logging related?

Reply:
moved to train.